Essentials/src/main/java/com/earth2me/essentials/commands/Commandtpaccept.java

```diff
@@ -18,60 +24,105 @@ public Commandtpaccept() {
 
     @Override
     public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
-        final User requester;
-        try {
-            requester = ess.getUser(user.getTeleportRequest());
-        } catch (final Exception ex) {
+        final boolean excludeHere;
```

Comment:
This looks like a flag for wildcards/accept all requests? Might be more appropriate to name it as such (best I can think of is `acceptAll`, but there's probably a better name)